box2d.pro

```diff
@@ -9,8 +9,13 @@ OBJECTS_DIR = .obj
 
 contains(QT_CONFIG, reduce_exports): CONFIG += hide_symbols
 
-INCLUDEPATH += .
-include(Box2D/box2d.pri)
+# uncomment the line below to compile qml_box2d plugin with Box2D library, installed in OS
+# warning: Box2D library must be already installed in system, for example in Debian it could be done with:
+# sudo apt-get install libbox2d2.3.0 libbox2d-dev
+
+#DEFINES += BOX2D_SYSTEM
```

Comment:
Instead of uncommenting this, I think it is also possible to do:
```
qmake DEFINES+=BOX2D_SYSTEM
```
It would be good to mention this as well (when it is confirmed to work).